pkg/cluster/client.go

```diff
@@ -292,6 +299,7 @@ type Options struct {
 	CrtClient              CrtClient
 	DiscoveryClientFactory DiscoveryClientFactory
 	DynamicClientFactory   DynamicClientFactory
+	TimeOut                time.Duration
```

Comment:
Might be better to use something like `RequestTimeout` so that users understand what the Timeout means here. I would also suggest adding a comment to explain the same.
```suggestion
	RequestTimeout                time.Duration
```